open-sphere-base/core/src/main/java/io/opensphere/core/util/swing/GenericFontIcon.java

```diff
@@ -28,7 +28,7 @@
     private static final int DEFAULT_SIZE = 16;
 
     /** A lock used to enforce synchronization. */
-    private static final Object PAINT_LOCK = new Object[0];
+    private final Object PAINT_LOCK = new Object[0];
```

Comment:
Doesn't removing the static nature of the lock object allow for the creation of a lock per instance of the class, which would be contrary to what we want for concurrency protection?

Reply:
The image rendering is per-instance of the GenericFontIcon class, so I think a static lock enforces unnecessary synchronization when rendering multiple icons. It doesn't make sense to me, but then I also am not entirely sure if Java uses the same Graphics2D space for every image. I think if it does, it shouldn't when using `createGraphics()`.